src/presenters/admin/indexation-permalink-warning-presenter.php

```diff
@@ -16,14 +16,33 @@
  */
 class Indexation_Permalink_Warning_Presenter extends Indexation_Warning_Presenter {
 
+	/**
+	 * Represents the reason that the permalink settings are changed.
+	 */
+	const REASON_PERMALINK_SETTINGS   = 'permalink_settings_changed';
+
+	/**
+	 * Represents the reason that the category base is changed.
+	 */
+	const REASON_CATEGORY_BASE_PREFIX = 'category_base_changed';
+
```

Comment:
It seems strange to me to have these constants defined in a presenter.
Though there is not a data layer behind this I suppose. The watcher is worse because you have the two different ones.
Maybe the indexation integration, but I'm not convinced that's better.

Reply:
I agree. I have no proper other place for this. I keep it this way.